docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -0,0 +1,197 @@
+
+**Step 6. Sign organization creation transactions**
+
+Now you need to sign two transactions in MetaMask to create your organization. Open MetaMask if the wallet window does not open automatically. Set the gas limit for each transaction at 6,660,000 to ensure that the transactions succeed. The second transaction may show up in MetaMask as a transaction with a blue dot next to it, indicating that it needs to be signed. Click the blue dot to open the transaction and sign it.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daomm1.png)
+
+**Step 7. Click “Get Started”**
+
+Your new Democracy DAO is ready to go!
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/orgready.png)
+
+### 2. Explore the Democracy DAO
+
+#### Home app
+
+The Home app welcomes you to your DAO and offers you several options for different actions you can take to interact with your DAO. You can also use the menu on the left-hand side of the screen to view the different apps that are currently installed for your DAO. Aragon Core currently has the Home, Token Manager, Voting, Finance, and Settings apps installed, with a Permissions and Apps Manager coming soon.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daohome.png)
+
+#### Token Manager app
+
+The Token Manager is the app you use to mint new tokens and assign them to yourself or other Ethereum accounts.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daotokenmanager.png)
+
+You can mint new tokens by following these steps:
+
+**Step 1. Click “Assign Tokens” button**
+
+Enter the address of the recipient of the tokens and the number of new tokens you want to mint and assign to that address, then click “Assign Tokens”.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daoaddmember.png)
+
+**Step 2. Create vote to assign tokens**
+
+Even though you are the only tokenholder, the Democracy DAO still requires a tokenholder vote for every action. Click “Sign Transaction” to sign the transaction that creates a new vote to assign the new token you want to mint. 
+
+Note: Creating a vote will currently automatically make your account vote “Yes” on the proposal. You can change your vote by opening the vote in the Voting app, clicking the “No” button, and signing the transaction to change your vote to “No”.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daoaddmembervote.png)
+
+**Step 3. Sign and send transaction to create the “Assign Tokens” vote**
+
+After clicking “Sign Transaction”, a notification will pop-up in MetaMask. Open MetaMask, then sign and send the transaction to create the vote.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daoaddmembervotesign.png)
+
+Step 4. Check out the newest member of your DAO!
```

Comment:
Use ##### to create a menu sub-item for easier navigation

Needs bold

Reply:
Bold and #### heading would be redundant